packages/cardano-services/test/jest-setup/rebuild-test-db.sh

```diff
@@ -7,9 +7,11 @@ WORKSPACE_ROOT="$(dirname "${PACKAGES_DIR}")"
 SECRETS_DIR="$PACKAGES_DIR"/cardano-services/placeholder-secrets
 
 yarn --cwd "$PACKAGES_DIR"/e2e local-network:down
-yarn --cwd "$PACKAGES_DIR"/e2e local-network:up -d
+yarn --cwd "$PACKAGES_DIR"/e2e local-network:up -d --build
 yarn --cwd "$WORKSPACE_ROOT" build
 yarn --cwd "$PACKAGES_DIR"/e2e test:wallet
+yarn --cwd "$PACKAGES_DIR"/e2e test:long-running --forceExit
```

Comment:
This is no longer necessary as we [added --forceExit to package.json script](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/e2e/package.json#L20)